models/admin.go

```diff
@@ -60,6 +62,23 @@ func RemoveAllWithNotice(title, path string) {
 	removeAllWithNotice(x, title, path)
 }
 
+func removeAllFromBucket(bucketPath, objKey string) error {
+	ctx := context.Background()
+	bucket, err := setting.OpenBucket(ctx, bucketPath)
+	if err != nil {
+		return fmt.Errorf("could not open bucket: %v", err)
```

Comment:
Is there a way to make an error here gracefully fail, so the user can attempt to retry the operation? Third-party requests are likely to fail at some point.

Reply:
The error related to `OpenBucket` is handled by the bucket providers using `exponential backoff` algo. I think it's okay not to handle it here.